v1.0/backend/data/schools.json

```diff
@@ -8,7 +8,8 @@
     "autoSyncFrequency": 600000,
     "storeTrainingData": true,
     "tags": true,
-    "supportEmail": "abc@gmail.com , xyz@gmail.com"
+    "supportEmail": "abc@gmail.com , xyz@gmail.com",
+    "experimentalOMRDetection": true
```

Comment:
@navin9584 / @apurvabisht97 , no school-level configurations is needed for this. This flag is needed only at layout json level.